inc/post-social.php

```diff
@@ -88,7 +88,13 @@ function largo_post_social_links( $echo = true ) {
 		}
 		
 		if ( $utilities['email'] === '1' ) {
-			$output .= '<span data-service="email" class="email custom-share-button share-button"><a><i class="icon-mail"></i> <span class="hidden-phone">' . esc_attr( __( 'Email', 'largo' ) ) . '</span></a></span>';
+			$output .= sprintf(
+				'<span data-service="email" class="email share-button"><a href="mailto:?subject=%2$s&body=%3$s%0D%0A%4$s" target="_blank"><i class="icon-mail-alt"></i> <span class="hidden-phone">%1$s</span></a></span>',
```

Comment:
Just a quick note on this - 

The body field still opens in Apple Mail with the post URL appended to the end of the excerpt (no space in-between), but I don't believe there's any way around this.

We could add a `<br />` tag, but that's not the correct way to do that, and it would likely have mixed results in browser-driven email applications.

For more info, see http://stackoverflow.com/questions/22765834/insert-a-line-break-in-mailto-body#22765878